homeassistant/components/device_tracker/__init__.py

```diff
@@ -486,7 +486,7 @@ def get_vendor_for_mac(self):
             return None
 
         # prevent lookup of invalid macs
-        if not len(self.mac.split(':')) == 6:
+        if not len(self.mac) == 17:
```

Comment:
What happens now for other delimeters? Like `-`? Or no delimeters?

Maybe you should combine the two tests? Or simply fail silently if macvendor blows up?